### PR TITLE
Add Release Notes for 1.4.2

### PR DESCRIPTION
**Pull request by rodolforg**

### CODE REVIEW

**Comment by morevnaproject** [comment]
@rodolforg Thank you for preparing the list of features for Release Notes! 

I have reorganized the list and removed items which are not noticeable for user. Will continue by polishing explanations and adding some screenshots.

**Comment by rodolforg** [comment]
@morevnaproject  no problem at all. :) I was just listing anything that was merged to the PR/branch.

I think it would be a good practice to keep. LibreOffice does it. The [current officially released versions are 7.0.x and 7.1.x ](https://www.libreoffice.org/download/download/), but they already write partial release notes for [7.2]( https://wiki.documentfoundation.org/ReleaseNotes/7.2) and [7.3]( https://wiki.documentfoundation.org/ReleaseNotes/7.3).

File: source/releases/stable/1.4.2.rst [hunk 1]
```diff
@@ -6,12 +6,12 @@ Synfig 1.4.2 Release Notes
...
 Synfig Studio 1.4.2 delivers some improvements and critical bugfixes. Please read below for details.
 
-Improved Lottie Exporter plugin
-----------------------------------------
+Improved export for Web
+------------------------
 
-* Added support for exporting Advanced Outline layer (`#2199 <https://github.com/synfig/synfig/pull/2199>`_).
-* Added support for exporting Blur layers (`#1505 <https://github.com/synfig/synfig/pull/1505>`_).
-* Added support for exporting Outline layer without variable widths (`#1647 <https://github.com/synfig/synfig/pull/1647>`_).
```
Comment (rodolforg) [inline source/releases/stable/1.4.2.rst]: I don't think it's not available: I believe it wasn't a feature export "without variable width", it was a constraint, that was solved by "Added support for exporting Advanced Outline layer (`#2199 <https://github.com/synfig/synfig/pull/2199>`_)."

Comment (morevnaproject) [reply source/releases/stable/1.4.2.rst]: No, this is a different feature, related with ignoring any line width, for any outline layer - https://github.com/synfig/synfig/pull/1647
This feature was requested by me in this issue - https://github.com/synfig/synfig/issues/1217

Currently the code has this feature included, but it is not possible to enable this mode from UI - it is available in [extended plugin system](https://github.com/synfig/synfig/pull/1647#issuecomment-679846675), which is not in 1.4.x branch yet. ^__^

**Pull request merged**